src/js/angular/controllers.js

```diff
@@ -135,12 +135,37 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
     $scope.descr = $translate.instant('main.gdb.description');
     $scope.documentation = '';
     $scope.menu = $menuItems;
+    $scope.menuCollapsed = false;
     $scope.tutorialState = LocalStorageAdapter.get(LSKeys.TUTORIAL_STATE) !== 1;
     $scope.userLoggedIn = false;
     $scope.embedded = $location.search().embedded;
     $scope.productInfo = productInfo;
     $scope.guidePaused = 'true' === LocalStorageAdapter.get(GUIDE_PAUSE);
 
+    $scope.isMenuCollapsedOnLoad = function () {
+        return $('.main-menu').hasClass('collapsed');
+    };
+
+    $scope.checkMenu = debounce(function () {
+        const collapsed = $scope.isMenuCollapsedOnLoad();
+        if ($scope.menuCollapsed !== collapsed) {
```

Comment:
this if statement is redundant, this can be just `$scope.menuCollapsed = collapsed`